core/templates/local_vector.h

```diff
@@ -279,6 +279,13 @@ class LocalVector {
 		}
 	}
 
+	void append_array(Span<T> p_span) {
+		reserve(p_span.size());
```

Comment:
Don't you mean `reserve(size() + p_span.size());` ?

Reply:
Oh, shoot. Even though I talk about this quirk a lot lately, the second I call `reserve`, I make this mistake myself 😅